build_defs/go.build_defs

```diff
@@ -1360,13 +1363,15 @@ def go_module(name:str='', module:str, version:str='', download:str='', deps:lis
 
 
 def _go_pkg_info(name:str, srcs:list, importconfig:str=None, import_path:str="", package:str="",
-                 strip:bool=False, module:str="", complete:bool=False, test_only:bool=False):
+                 strip:bool=False, module:str="", complete:bool=False, test_only:bool=False, install:list=[]):
     """Internal-only function for generating the pkg_info files"""
     cflag = '-c' if complete else ''
     sflag = '-s "$SRCS_SRCS"' if strip else ''
 
+    install_flags = ' '.join([f'-p {i}' for i in install if i != ""])
```

Comment:
nit: would just write `for i in install if i`, the `!= ""` is implicit

Reply:
Oh good point, thanks!